server/publications/collections/packages.js

```diff
@@ -102,7 +102,14 @@ Meteor.publish("Packages", function (shopId) {
       }
       // observe and transform Package registry adds i18n and other meta data
       const observer = Packages.find({
-        shopId: myShopId
+        $or: [
+          { shopId: myShopId },
+          {
+            name: {
+              $in: ["reaction-social"]
```

Comment:
This definitely seems like the wrong approach here. We shouldn't need to make a change to this global query for just this one package.